test/kwokctl/metric.yaml

```diff
@@ -22,30 +22,108 @@ spec:
       buckets:
         - le: 0.005
           value: '0'
-
         - le: 0.01
-          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 10'
+          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 10.0'
         - le: 0.025
-          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 9'
+          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 9.0'
         - le: 0.05
-          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 9'
+          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 9.0'
         - le: 0.1
-          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 8'
+          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 8.0'
         - le: 0.25
-          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 8'
+          value: '(unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)) / 8.0'
         - le: 0.5
           value: 'unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)'
         - le: 1
           value: 'unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)'
         - le: 2.5
           value: 'unixSecond(now()) - unixSecond(node.metadata.creationTimestamp)'
         - le: 5
-          value: '1'
+          value: '1.0'
         - le: 7.5
-          value: '1'
+          value: '1.0'
           hidden: true
         - le: 10
-          value: '1'
+          value: '1.0'
         - le: 20
-          value: '1'
+          value: '1.0'
           hidden: true
+---
+kind: Metric
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: fake-node
+spec:
+  path: "/metrics/resource/nodes/fake-node"
+  metrics:
+    - name: container_cpu_usage_seconds_total
+      dimension: container
+      help: "[ALPHA] Cumulative cpu time consumed by the container in core-seconds"
+      kind: counter
+      labels:
+        - name: container
+          value: 'container.name'
+        - name: namespace
+          value: 'pod.metadata.namespace'
+        - name: pod
+          value: 'pod.metadata.name'
+      value: 'pod.sinceSecond() / 100.0'
```

Comment:
https://github.com/kubernetes-sigs/kwok/pull/669/files#r1241220124

As I mentioned earlier, the `pod.sinceSecond()` function here may be confusing. - Where does it come from?

Reply:
`pod.sinceSecond()` same as `sinceSecond(pod)`